extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,46 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live stories
+
+If the `live-story` attribute is specified on the `<amp-story>` element, it will allow users to receive real-time updates coming from the server. It updates the client device without having to refresh the page and shows a notification when there are updates available and the user is on the last page of the story.
+
+<figure class="centered-fig">
+  <amp-anim alt="Live stories example" width="300" height="533" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/live-stories-gif.gif">
+  <noscript>
+    <img alt="Live stories example" width="200" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/live-stories-gif.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+**Note**: As long as the `live-story` attribute is present on the `<amp-story>` element, the client will make continous polls to the server. Make sure to set the `live-story-disabled` attribute to make the polling stop once you're finished with the live broadcast of your story.
+
+Usage:
+* Import the `amp-live-list` extension in the `<head>` tag of your document.
+* Specify an `id` on the `<amp-story>` element.
+* Add the `live-story` attribute to the `<amp-story>` element.
+* [Optional] Add the [`data-poll-interval`](#data-poll-interval-[optional]) attribute to the `<amp-story>` element to specify a time interval for checking for new updates.
+* [Optional] Add the [`live-story-disabled`](#live-story-disabled-[optional]) attribute to the `<amp-story>` element to disable the polling.
+* On each `<amp-story-page>`:
+  * Specify a `data-sort-time` attribute with a valid value. This is a timestamp used for sorting entries. Higher timestamps will be inserted before older entries. We recommend using [Unix time](https://www.unixtimestamp.com/).
+  * [Optional] Specify the `data-update-time` attribute with a valid value. This is a timestamp when the entry was last updated. Use this attribute to trigger an update on an existing page: the client will replace all existing content in this page with the new, updated content. We recommend using [Unix time](https://www.unixtimestamp.com/).
```

Comment:
Same question, do we actually replace the pages?

Reply:
Discussed offline - this is handled by the amp-live-list code.